Common/src/CConfig.cpp

```diff
@@ -5175,6 +5183,35 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
       nTurbVar = 2; break;
   }
 
+  /*--- Postprocess SST_OPTIONS into structure. ---*/
+  //TODO this wont work.
+  string sst_string = "SST";
+  struct SST_Struct {
```

Comment:
declare and define the struct in option_structure, close to SST_OPTIONS.
Then create a free function in there that populates the struct based on the SST_Options array (you pass the pointer and size to the function and it returns the struct.
Here in CConfig you put a member variable of the type of that struct, e.g. SST_ParsedOptions, and then do:
SST_ParsedOptions = ParseSSTOptions(SST_Options, nSST_Options);
Finally you have a method to return this SST_ParsedOptions to use it in sources, etc.